src/CG/CG_elem.cpp

```diff
@@ -2113,6 +2113,829 @@ bool intersectSegmentPolygon( array3D const &P0, array3D const &P1, std::size_t
     return false;
 }
 
+/*!
+ * Check if a pixel is crossing a given plane and fully calculate the intersection (a segment) with 
+ * edges, eventually. A pixel is defined as a planar rectangular, vertical to a cartesian axis.
+ * The polygon greated from the intersection belongs to the subspace opposite of the
+ * one pointed by the plane's normal
+ *
+ * Given the normal cartesian direction dir, the pixel coordinates
+ * are represened on its 2D plane as (x, y) where x = (dim + 1) % 3 and y = (dim + 2) % 3 
+ *
+ * Local indeces for edges are defined as follows:
+ * edge 0: y = y min
+ * edge 1: x = x max
+ * edge 2: y = y max
+ * edge 3: x = x min
+ *
+ * Co-planarity of input pixel and plane will produce no intersection.
```

Comment:
This seems not coherent with all other functions in this module.

Reply:
See my comment in a previous similar case.